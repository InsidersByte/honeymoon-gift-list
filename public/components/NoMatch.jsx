/* @flow */

import React from 'react';
import { Link } from 'react-router';
import { HOME_ROUTE } from '../constants/routes';

const styles = {
    root: {
        alignItems: 'center',
        display: 'flex',
        flexDirection: 'column',
        justifyContent: 'center',
        minHeight: '75%',
    },
    title: {
        marginBottom: 15,
    },
    content: {
        marginBottom: 20,
    },
};

export default function NoMatch() {
    return (
        <div style={styles.root}>
            <h1 style={styles.title}>Page not found</h1>

            <div style={styles.content}>
                Woops! Sorry, there is nothing to see here.
            </div>

            <Link to={HOME_ROUTE}>Back to Home</Link>
        </div>
    );
}
