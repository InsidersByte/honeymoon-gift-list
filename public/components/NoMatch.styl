.root
    display flex
    flex-direction column
    justify-content center
    align-items center
    min-height 75%

.title
    margin-bottom 15px

.content
    margin-bottom 20px
