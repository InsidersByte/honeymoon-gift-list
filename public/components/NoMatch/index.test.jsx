import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from './';

describe('NoMatch', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NoMatch />);
    expect(wrapper).toMatchSnapshot();
  });
});
