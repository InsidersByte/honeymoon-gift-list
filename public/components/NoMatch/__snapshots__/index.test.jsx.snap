// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
      "minHeight": "75%",
    }
  }
>
  <h1
    style={
      Object {
        "marginBottom": 15,
      }
    }
  >
    Page not found
  </h1>
  <div
    style={
      Object {
        "marginBottom": 20,
      }
    }
  >
    Woops! Sorry, there is nothing to see here.
  </div>
  <Link
    replace={false}
    to="/"
  >
    Back to Home
  </Link>
</div>
`;
